Parsers/ASimWebSession/Parsers/ASimWebSessionPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,176 @@
+Description: |
+  This ASIM parser supports normalizing Palo Alto Cortex Data Lake logs to the ASIM Web Session normalized schema. These events are captured through the Palo Alto Networks CDL data connector that ingests CDL logs into Microsoft Sentinel.
+ParserName: ASimWebSessionPaloAltoCortexDataLake
+EquivalentBuiltInParser: _ASim_WebSession_PaloAltoCortexDataLake
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventSeverityLookup = datatable (LogSeverity: string, EventSeverity: string)
+      [
+      "0", "Low",
+      "1", "Low",
+      "2", "Low",
+      "3", "Low",
+      "4", "Medium",
+      "5", "Medium",
+      "6", "Medium",
+      "7", "High",
+      "8", "High",
+      "9", "High",
+      "10", "High"
+  ];
+  let EventLookup=datatable(DeviceAction:string, DvcAction:string,EventResult:string,EventResultDetails:string)
+    [
+        "alert", "Allow", "Success","200", 
+        "allow", "Allow", "Success", "200",
+        "continue", "Allow", "Success", "200",
+        "override", "Allow", "Success", "200",
+        "block-continue", "Allow", "Partial", "200",
+        "block-url", "Deny", "Failure", "503",
+        "block-override", "Deny", "Failure", "302",
+        "override-lockout", "Deny", "Failure","503",
+        "reset client", "Reset Source", "Failure", "503",
+        "reset server", "Reset Destination", "Failure", "503",
+        "reset both", "Reset", "Failure", "503",
+        "deny", "Deny", "Failure", "503",
+        "drop", "Drop", "Failure", "503",
+        "drop ICMP", "Drop ICMP", "Failure", "503"
+    ];
+  let parser = (disabled: bool=false) {
+      CommonSecurityLog
+      | where not(disabled)
+          and DeviceVendor == "Palo Alto Networks" and DeviceProduct == "LF"
+          and DeviceEventClassID == "THREAT" and Activity == "url"
+      | parse-kv AdditionalExtensions as (PanOSDestinationUUID: string, PanOSDestinationLocation: string, PanOSDestinationDeviceMac: string, PanOSSourceUUID: string, PanOSSourceDeviceMac: string, PanOSReferer: string, PanOSIsClienttoServer:string, PanOSSourceDeviceHost: string, PanOSDestinationDeviceHost:string, start: string) with (pair_delimiter=";", kv_delimiter="=")
+      | invoke _ASIM_ResolveDvcFQDN('DeviceName')
+      | invoke _ASIM_ResolveSrcFQDN('PanOSSourceDeviceHost')
+      | invoke _ASIM_ResolveDstFQDN('PanOSDestinationDeviceHost')
+      | lookup EventSeverityLookup on LogSeverity
+      | lookup EventLookup on DeviceAction
+      | extend
+          EventStartTime = todatetime(start),
+          SrcIpAddr = coalesce(DeviceCustomIPv6Address2, SourceIP),
+          DstIpAddr = coalesce(DeviceCustomIPv6Address3, DestinationIP),
+          HttpRequestMethod = toupper(RequestMethod),
+          NetworkProtocol = toupper(Protocol),
+          NetworkSessionId = tostring(FieldDeviceCustomNumber1),
```

Comment:
Can you validate this?
I think SessionID would be DeviceCustomNumber1 instead of FieldDeviceCustomNumber1
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/9af029f2-8d11-4a6c-bbf2-cc83b49043ba)

Reply:
DeviceCustomNumber1 field is soon to be deprecated field and replaced by FieldDeviceCustomNumber1 as given in Microsoft document of CommonSecurityLog table.
![image](https://github.com/Azure/Azure-Sentinel/assets/107753570/8e9ebc0d-305f-492e-bd5d-34447174d4c6)

Reply:
nice. thanks